example/lib/src/features/home/views/ui_states/toast_code_formatter.dart

```diff
@@ -18,7 +18,7 @@ class ToastCodeFormatter {
     title: '${toastDetail.title}',
   ''');
 
-    if (toastDetail.description.isNotEmpty) {
+    if (toastDetail.description != null) {
       code.writeln('\tdescription: \'${toastDetail.description}\',');
```

Comment:
Are you sure is this working? 

I think you should change it to this
``` dart
code.writeln('\tdescription: Text(\'${toastDetail.description}\'),');
```

Reply:
Also for the title

Reply:
I will try this out thanks!